api/src/wfl/stage.clj

```diff
@@ -0,0 +1,30 @@
+(ns wfl.stage
```

Comment:
I agree that the name stage is clunky but can provide no better suggestions. Thanks for calling it out :)

Reply:
you could call it processing-stage then import `:as stage` if you're worries about much typing.

Reply:
My namespace aliasing preference is to avoid renaming unless there's an established convention (ex. `[clojure.spec.alpha :as s]`).  This [blog post](https://stuartsierra.com/2015/05/10/clojure-namespace-aliases) does a good job of summing up why.

I don't mind more typing as a tradeoff for better readability, but a longer alias tells us that we have an opportunity for better code organization.

Is there an argument against adding an additional directory layer to group queue-based pipeline management code, or are we just stuck on naming?

Reply:
I just think that's overkill

Reply:
Okay.  I will leave as is for now, and we can add an additional layer without much effort later if it seems appropriate.